flyteadmin_config.yaml

```diff
@@ -64,7 +64,7 @@ database:
   port: 5432
   username: postgres
   host: localhost
-  dbname: postgres
+  dbname: flyteadmin
```

Comment:
Until we fix this: https://github.com/flyteorg/flyte/issues/1882, I don't think we should attempt to use anything but the default DB...

Reply:
Ok.reverted this to older values of postgress